### PR TITLE
Add program for cleaning up maintenance_release projects for containers

### PR DESCRIPTION
**Pull request by Vogtinator**
Currently every single container released to `openSUSE:Containers:Tumbleweed` is kept forever, that needs to change.

Only those containers providing binaries to the latest five version for each
archs are kept, all others are deleted.

### CODE REVIEW

File: gocd/container-cleaner.gocd.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+format_version: 3
+pipelines:
+  ContainerCleaner.Tumbleweed:
+    group: openSUSE.Checkers
+    lock_behavior: unlockWhenFinished
+    environment_variables:
+      OSC_CONFIG: /home/go/config/oscrc-totest-manager
+    materials:
+      script:
+        git: https://github.com/openSUSE/osc-plugin-factory.git
```
Comment (coolo) [inline gocd/container-cleaner.gocd.yaml]: better use the proper name of the repo

**Comment by Vogtinator** [comment]
Not sure why, but apparently the used timer spec `0 0-23/2 * ? * *` results in it running every 2nd minute.

**Comment by coolo** [comment]
and you intentended hours? Then move it down one more ;)

https://docs.gocd.org/current/configuration/configuration_reference.html#timer

**Comment by Vogtinator** [comment]
Apparently I read the wrong documentation. Fixed.

**Pull request merged**